contracts/network/BancorNetwork.sol

```diff
@@ -315,11 +320,11 @@ contract BancorNetwork is IBancorNetwork, Upgradeable, ReentrancyGuardUpgradeabl
         // set up administrative roles
         _setRoleAdmin(ROLE_MIGRATION_MANAGER, ROLE_ADMIN);
         _setRoleAdmin(ROLE_EMERGENCY_STOPPER, ROLE_ADMIN);
-        _setRoleAdmin(ROLE_NETWORK_FEE_MANAGER, ROLE_ADMIN);
 
         _depositingEnabled = true;
 
         _setPOLRewardsPPM(2000);
+        _setMinNetworkFeeBurn(1_000_000e18);
```

Comment:
Since this also needs to be called during an upgrade, I'd consider adding the upgrade deployment script to this PR as well

Reply:
Added the upgrade script